action.yml

```diff
@@ -98,3 +100,4 @@ runs:
         INPUT_GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         python ${{ github.action_path }}/main.py
+      shell: bash
```

Comment:
Should not define default shell. Let it for the user

Reply:
I agree, but 'shell' parameter is required in action.yml.